lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -487,12 +487,27 @@ export abstract class BaseQueryBuilderService {
         let query = '';
 
         this.categories.forEach((facet) => {
-            const customQuery = this.queryFragments[facet.id];
-            if (customQuery) {
-                if (query.length > 0) {
-                    query += ' AND ';
+            let customQuery: string;
+
+            if (facet.component.selector === 'widget-composite') {
```

Comment:
We do we need this change? Parent component using the tabbed component should update both displayValue and query on its own. It should produce just one combined query for whole component, I don't think we need any changes in query builder

Reply:
This is leftover code from a previous approach that has since been scrapped. I've reverted this change back to its original state